larky/src/main/resources/vgs/http/response.star

```diff
@@ -0,0 +1,47 @@
+load("@stdlib//larky", larky="larky")
+load("@stdlib//urllib/parse", "parse")
+
+def VGSHttpResponse(
+    body=None,
+    headers={},
+    status_code=200
+):
+
+    self = larky.mutablestruct(__name__="VGSHttpResponse", __class__=VGSHttpResponse)
+
+    # body property
+    def _get_body():
+        return self._body
+
+    def _set_body(body):
+        if body != self._body:
+            self._body = body
+            # issue 16464
+            # if we change data we need to remove content-length header
+            # (cause it's most probably calculated for previous value)
+            if "Content-length" in self.headers:
+                self.remove_header("Content-length")
+    self.body = larky.property(_get_body, _set_body)
+
+    def add_header(key, val):
+        self.headers[key] = val
+    self.add_header = add_header
+
+    def __init__(
+        body=None,
+        headers={},
```

Comment:
no point in having these defaults be in `__init__` if they're already defined here: https://github.com/verygoodsecurity/starlarky/pull/244/files#diff-1d171479d8b2520c1c1417a2bf8e5f8f6c9fdcc448bae315827aff445b3064a6R4